shapeshift/web
# fix: save the mobile wallet only if the seed phrase has been confirmed

## Description
The mobile wallet is created after you submit the name and see the seed phrase instead of creating it when you confirm the seed phrase

I move the `addWallet` after validating the seed phrase instead of when you see the seed for the first time



## Notice




## Pull Request Type


## Issue (if applicable)



closes #2732

## Risk
Mobile Wallet creating could be buggy somehow


## Testing
- Go to `Connect wallet` on the mobile app
- Go to `Shapeshift Mobile`
- `Create wallet`
- Enter a name, show the seed, save
- Go back 2 times
- There should be no saved wallets

- `Create wallet`
- Enter a name, show the seed, save
- Close the modal
- Open the `Shapeshift mobile` wallet view again, there should be no saved wallets

Try to complete every required steps, and you should have a saved wallet 



### Engineering
See testing


### Operations
See testing



## Screenshots (if applicable)
